outport/process/factory/check.go

```diff
@@ -1,12 +1,17 @@
 package factory
 
 import (
+	"errors"
+
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	"github.com/ElrondNetwork/elrond-go/outport/process"
 	"github.com/ElrondNetwork/elrond-go/outport/process/alteredaccounts"
 	"github.com/ElrondNetwork/elrond-go/outport/process/transactionsfee"
 )
 
+// ErrNilHasher signals that a nil hasher has been provided
+var ErrNilHasher = errors.New("nil hasher provided")
```

Comment:
Why didn't put this in outport/process/errors.go ?

Reply:
moved.